content/en/docs/4.directory-structure/12.store.md

```diff
@@ -192,7 +192,7 @@ And the corresponding actions can be in the file  `store/actions.js`
 
 ```js{}[store/actions.js]
 export default {
-  async fetchCounter(state) {
+  async fetchCounter(context) {
     // make request
     const res = { data: 10 };
     state.counter = res.data;
```

Comment:
Good catch. We probably also need to update this line.

Reply:
I think it's already in the PR, isn't it?

Reply:
Nope.

Reply:
@l-portet could you change the line 198 `context.counter` instead of `state.counter`

Reply:
Thanks @hadi-ilies!